packages/webpack/lib/utils/compiler-fork.js

```diff
@@ -0,0 +1,66 @@
+const { join } = require('path');
+const { serializeError } = require('serialize-error');
+
+const { configure } = require('../../');
+const { BuildError, CompilerError } = require('../utils/errors');
+const { createCompiler } = require('./compiler');
+
+process.on('message', (message) => {
```

Comment:
There is no need for it, I just preferred it. But why not?

Reply:
I think the logical unit of "starting" / "forking" a compiler and subscribing to its results encapsulates the forked script and therefore it makes sense to keep it all together. But in the end it's not a blocker for me.

Reply:
Ok, I tried but the actual reason is that we have a circular dependency if we don't move this into a new file because we use the compiler in the render middleware now.

`webpack/index.js` -> `webpack/lib/middlewares/render.js` -> `webpack/lib/utils/compiler` -> `webpack/index.js`.

Reply:
Oh, okay then :)